Readme.md

```diff
@@ -85,6 +85,12 @@ port = 12201
 level = "INFO"
 ```
 
+# Optional logging config, to log to stderr
```

Comment:
This is outside of the `toml` code block. It should be inside. Easy to misready the syntax here, but the `#` does not introduce a new headline, but is supposed to be a `toml` comment. Just look at the rendered markdown in the PRs branch. It'll become obvious immediatly.